v2/swiftui/App/Views/OpenRealmView.swift

```diff
@@ -0,0 +1,66 @@
+import SwiftUI
+import RealmSwift
+
+/// Called when login completes. Opens the realm asynchronously and navigates to the Items screen.
+struct OpenRealmView: View {
+    @AsyncOpen(appId: theAppConfig.appId, timeout: 2000) var asyncOpen
+    // We must pass the user, so we can set the user.id when we create Item objects
+    @State var user: User
+    @State var showMyItems = true
+    @State var syncEnabled = true
+    // Configuration used to open the realm.
+    @Environment(\.realmConfiguration) private var config
+
+    var body: some View {
+        switch asyncOpen {
+            // Starting the Realm.asyncOpen process.
```

Comment:
Very very minor but for some reason I think all of these comments that are above the `case` actually belong below the case.